### ISSUE TITLE
Scheduler interval calculation AFUNAD

### ISSUE THREAD

**Issue by barbequesauce** [barbequesauce/Watcher3#32]
vis. scheduler.py, lines 59-64. This would make the scheduled time invalid if the minute is past the bottom of the hour.

Similar problems may exist in other scheduler lines

```
        conf = core.CONFIG['Postprocessing']['Scanner']
        interval = conf['interval'] * 60

        now = datetime.datetime.today()
        hr = now.hour
        minute = now.minute + interval
```

**Comment by barbequesauce** [comment]
Potential issues: 

```
app.py:183:                               'interval': obj.interval,
log.py:33:    file_handler = logging.handlers.TimedRotatingFileHandler(logfile, when='D', interval=1, backupCount=core.CONFIG['Server']['keeplog'], encoding='utf-8')
config.py:229:            interval = core.CONFIG['Server']['checkupdatefrequency'] * 3600
config.py:231:            core.scheduler_plugin.task_list['Update Checker'].reload(hr, min, interval, auto_start=auto_start)
config.py:238:            interval = core.CONFIG['Search']['rsssyncfrequency'] * 60
config.py:240:            core.scheduler_plugin.task_list['Movie Search'].reload(hr, min, interval, auto_start=auto_start)
config.py:247:                interval = core.CONFIG['Search']['Watchlists']['imdbfrequency'] * 60
config.py:249:                core.scheduler_plugin.task_list['IMDB Sync'].reload(hr, min, interval, auto_start=auto_start)
config.py:254:                interval = 24 * 3600
config.py:256:                core.scheduler_plugin.task_list['PopularMovies Sync'].reload(hr, min, interval, auto_start=auto_start)
config.py:261:                interval = core.CONFIG['Search']['Watchlists']['traktfrequency'] * 60
config.py:263:                core.scheduler_plugin.task_list['Trakt Sync'].reload(hr, min, interval, auto_start=auto_start)
config.py:267:        if any(i in d for i in ('interval', 'enabled')):
config.py:269:            min = now.minute + core.CONFIG['Postprocessing']['Scanner']['interval']
config.py:270:            interval = core.CONFIG['Postprocessing']['Scanner']['interval'] * 60
config.py:272:            core.scheduler_plugin.task_list['PostProcessing Scan'].reload(hr, min, interval, auto_start=auto_start)
config.py:279:            interval = 24 * 3600
config.py:281:            core.scheduler_plugin.task_list['Missing Files Scan'].reload(hr, min, interval, auto_start=auto_start)
scheduler.py:60:        interval = conf['interval'] * 60
scheduler.py:64:        minute = now.minute + interval
scheduler.py:66:        SchedulerPlugin.ScheduledTask(hr, minute, interval, PostProcessingScan.scan_directory, auto_start=conf['enabled'], name='PostProcessing Scan')
scheduler.py:137:        interval = core.CONFIG['Search']['rsssyncfrequency'] * 60
scheduler.py:143:        SchedulerPlugin.ScheduledTask(hr, min, interval, search.search_all, auto_start=True, name='Movie Search')
scheduler.py:151:        interval = 72 * 60 * 60  # 72 hours
scheduler.py:158:        SchedulerPlugin.ScheduledTask(hr, min, interval, MetadataUpdate.metadata_update, auto_start=True, name='Metadata Update')
scheduler.py:201:        interval = core.CONFIG['Server']['checkupdatefrequency'] * 3600
scheduler.py:214:        SchedulerPlugin.ScheduledTask(hr, min, interval, AutoUpdateCheck.update_check, auto_start=auto_start, name='Update Checker')
scheduler.py:240:        interval = core.CONFIG['Search']['Watchlists']['imdbfrequency'] * 60
scheduler.py:251:        SchedulerPlugin.ScheduledTask(hr, min, interval, imdb.sync, auto_start=auto_start, name='IMDB Sync')
scheduler.py:259:        interval = 24 * 3600
scheduler.py:269:        SchedulerPlugin.ScheduledTask(hr, min, interval, popularmovies.sync_feed, auto_start=auto_start, name='PopularMovies Sync')
scheduler.py:278:        interval = core.CONFIG['Search']['Watchlists']['traktfrequency'] * 60
scheduler.py:294:        SchedulerPlugin.ScheduledTask(hr, min, interval, trakt.sync, auto_start=auto_start, name='Trakt Sync')
scheduler.py:303:        interval = 24 * 3600
scheduler.py:310:        SchedulerPlugin.ScheduledTask(hr, min, interval, Manage.scanmissingfiles, auto_start=auto_start, name='Missing Files Scan')
base_config.cfg:152:            "interval": 4,
```

**Comment by barbequesauce** [comment]
```
>>> now = datetime.datetime.today()
>>> interval = 30
>>> hr = now.hour
>>> minute = now.minute + interval
>>> print('now is {} // hour is {} // minute is {} // interval is {}'.format(now, hr, minute, interval))
now is 2019-09-18 13:50:54.846811 // hour is 13 // minute is 80 // interval is 30

```

**Comment by barbequesauce** [comment]
Changes to fix #24 wound up making this go away.
